src/core_plugins/metrics/public/directives/visualization.js

```diff
@@ -0,0 +1,42 @@
+import _ from 'lodash';
+import $ from 'jquery';
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import Visualization from '../components/vis_editor/visualization';
+import addScope from '../lib/add_scope';
+import modules from 'ui/modules';
+import moment from 'moment';
+import createBrushHandler from '../lib/create_brush_handler';
+const app = modules.get('apps/metrics/directives');
+app.directive('metricsVisualization', (timefilter) => {
+  return {
+    restrict: 'E',
+    link: ($scope, $el, $attrs) => {
+      const addToState = ['model', 'visData', 'backgroundColor'];
+      const Component = addScope(Visualization, $scope, addToState);
+      const handleBrush = createBrushHandler($scope, timefilter);
+      render(<Component onBrush={handleBrush} className="dashboard__visualization"/>, $el[0]);
+      $scope.$on('$destroy', () => unmountComponentAtNode($el[0]));
+
+      // For Metrics, Gauges and markdown visualizations we want to hide the
+      // panel title because it just doens't make sense to show it.
+      const panel = $($el[0]).parents('.panel');
+      if (panel.length) {
+        const panelHeading = panel.find('.panel-heading');
+        const panelTitle = panel.find('.panel-title');
+        const matchingTypes = ['metric', 'gauge', 'markdown'];
+        if (panelHeading.length && panelTitle.length && _.contains(matchingTypes, $scope.model.type)) {
```

Comment:
currently on dashboard chart rerenders when you mouse over (as size gets changed when edit buttons are added ... this doesnt happen if the title is there)

just setting visibility to hidden should solve it, but i am wondering do we really want to hide the title ?